examples/hello-lb-v2.yml

```diff
@@ -9,6 +9,8 @@ scheduler:
   elb_v2:
     # VPC id where the target group is located
     vpc_id: vpc-WWWWWWWW
+    # If you want internal ELB, then use 'scheme'. (ex. internal service that like microservice inside VPC)
+  # scheme: internal
```

Comment:
ill-indented?

Reply:
I had thought that same indent to vpc_id.
however, it looks like misreadable indent. I did fixed it.